src-vue/src/views/RepairView.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="fc_repair__container">
+        <h2>Repair:</h2>
+
+        <h3>Northstar:</h3>
+
+        <el-button type="primary" @click="disableAllModsButCore">
+            Disable all but core mods
+        </el-button>
+
+        <h3>FlightCore:</h3>
```

Comment:
Both section headers ("_Northstar_" and "_FlightCore_") are `<h3>` but they render differently for some reason. No idea why :c

![image](https://user-images.githubusercontent.com/40122905/204100374-5d94e21b-268b-4a25-9939-e39362119d60.png)